perception/sub8_perception/CMakeLists.txt

```diff
@@ -202,3 +202,32 @@ target_link_libraries(
 install(
   TARGETS
 )
+
```

Comment:
Can you put this test cmake code in a CMakeLists.txt in the test dir and just include it here using [`add_subdirectory(path_to_testing_cmakelists)`](https://www.johnlamp.net/cmake-tutorial-4-libraries-and-subdirectories.html#section-TestingForReal)?

This is likely to get a lot larger and I'd like to be able to quickly skim thorough the programs that are getting compiled here in this CMakeLists.